rllib/core/learner/tf/tf_learner.py

```diff
@@ -137,6 +137,9 @@ def get_optimizer_obj(
         lr = self._optimizer_config["lr"]
         return optimizer_cls(learning_rate=lr)
 
+    def _is_module_compatible_with_learner(self, module: RLModule) -> bool:
+        return isinstance(module, tf.keras.Model)
```

Comment:
this learner class is really only compatible with TfRLModules, unless we stop using `TfRLModule.forward_train` in this class in favor of `__call__`

Reply:
fair.